src/main/java/EtfRecommendService/etf/domain/EtfReadProjection.java

```diff
@@ -0,0 +1,31 @@
+package EtfRecommendService.etf.domain;
+
+import EtfRecommendService.etf.Theme;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+@Entity
+public class EtfReadProjection {
```

Comment:
"Projection"이 이미 조회용임을 암시하기 때문에 아래처럼 줄여도 될 것 같아요
```suggestion
public class EtfProjection {
```